pkgs/tools/networking/fakeroute/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "fakeroute-${version}";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "https://moxie.org/software/fakeroute/${name}.tar.gz";
+    sha256 = "1sp342rxgm1gz4mvi5vvz1knz7kn9px9s39ii3jdjp4ks7lr5c8f";
+  };
+
+  meta = with stdenv.lib; {
+    description = ''
+      Makes your machine appear to be anywhere on the internet
+      to any host running a (UDP) unix traceroute
+    '';
+    homepage = https://moxie.org/software/fakeroute/;
+    #license = licenses.???;
```

Comment:
It's `licenses.bsd3`. In the source files you can see a license text on the top which is the [BSD 3-Clause License](https://opensource.org/licenses/BSD-3-Clause). Though the third clause is oddly numbered as fourth. :goat: